data/aviation_data.csv

```diff
@@ -0,0 +1,274 @@
+country_name,icao,icaoregion,statfor,pru
```

Comment:
I see there are other PR, i.e. the one with official UN country short names, and I did want to make the process to add specific extensions easier in case this will not get into the official release.
Please let me know whether you want it merged.